wizeline/wize-q-remix
# fix: Departments filter overflow

#### What does this PR do?
- Fix dropdown list not scrollable bug when there is a filter selected on departments

#### How should this be manually tested?
1. Select a department filter which contains more than 6 questions
2. Scroll down
3. You should be able to see all filters on departments list

#### What are the relevant tickets?
Closes [wize-q-remix #135](https://github.com/wizeline/wize-q-remix/issues/135)

#### Screenshots

https://user-images.githubusercontent.com/45496427/210389748-ea12d499-1c5d-4e73-a3ed-03677fcd442e.mov
